ACE/bin/auto_run_tests.pl

```diff
@@ -31,20 +32,24 @@
     $DDS_ROOT = "$ACE_ROOT/TAO/DDS";
 }
 
-sub run_command ($$) {
+sub run_command {
+  my $test = shift;
   my $command = shift;
   my $print_error = shift;
+
   my $result = 0;
-  if (system($command)) {
+  if (system ($command)) {
     $result = $? >> 8;
     if ($print_error) {
+      my $signal = $? & 127;
+      my $coredump = $? & 128;
       my $error_message;
       if ($? == -1) {
         $error_message = "failed to run: $!";
       }
-      elsif ($? & 127) {
-        $error_message = sprintf("exited on signal %d", ($? & 127));
-        $error_message .= " and created coredump" if ($? & 128);
+      elsif ($signal) {
+        $error_message = sprintf ("exited on signal %d", ($signal));
```

Comment:
```suggestion
        $error_message = sprintf("exited on signal %d", $signal);
```
Not sure how strictly we care about Perl's official style (in the perlstyle man page), but ACE-guidelines says we should use it.  (There are a few more like this if you decide to change it.)